internal/test_helpers/fixtures/bash/history_persistence/pass

```diff
@@ -0,0 +1,162 @@
+[33m[stage-4] [0m[92m✓ Found command "echo orange raspberry" in history file[0m
+[33m[stage-4] [0m[92m✓ Found command "echo apple banana" in history file[0m
+[33m[stage-4] [0m[92m✓ Found command "echo strawberry pear" in history file[0m
+[33m[stage-4] [0m[92m✓ Found command "history -w /tmp/banana_shell_history_test" in history file[0m
+[33m[your-program] [0m$ 
+[33m[stage-4] [0m[92mTest passed.[0m
+
+[33m[stage-3] [0m[94mRunning tests for Stage #3: sx3[0m
+[33m[stage-3] [0m[94mOriginal history file content:[0m
+[33m[stage-3] [0m[grape_shell_history_test] echo strawberry pineapple blueberry
+[33m[stage-3] [0m[grape_shell_history_test] echo orange pineapple apple
+[33m[stage-3] [0m[grape_shell_history_test] echo pineapple apple mango
+[33m[stage-3] [0m[94mRunning ./your_shell.sh[0m
+[33m[your-program] [0m$ echo pineapple banana pear
+[33m[your-program] [0mpineapple banana pear
+[33m[stage-3] [0m[92m✓ Ran echo pineapple banana pear[0m
+[33m[your-program] [0m$ echo grape pear
+[33m[your-program] [0mgrape pear
+[33m[stage-3] [0m[92m✓ Ran echo grape pear[0m
+[33m[your-program] [0m$ history
+[33m[your-program] [0m    1  echo pineapple banana pear
+[33m[your-program] [0m    2  echo grape pear
+[33m[your-program] [0m    3  history
+[33m[stage-3] [0m[92m✓ History before appending is correct[0m
+[33m[your-program] [0m$ history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[92m✓ History appended to file[0m
+[33m[stage-3] [0m[94mHistory file content after appending:[0m
+[33m[stage-3] [0m[grape_shell_history_test] echo strawberry pineapple blueberry
+[33m[stage-3] [0m[grape_shell_history_test] echo orange pineapple apple
+[33m[stage-3] [0m[grape_shell_history_test] echo pineapple apple mango
+[33m[stage-3] [0m[grape_shell_history_test] echo pineapple banana pear
+[33m[stage-3] [0m[grape_shell_history_test] echo grape pear
+[33m[stage-3] [0m[grape_shell_history_test] history
+[33m[stage-3] [0m[grape_shell_history_test] history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[92m✓ History file has correct number of lines[0m
+[33m[stage-3] [0m[92m✓ Found initial command "echo strawberry pineapple blueberry" in history file[0m
+[33m[stage-3] [0m[92m✓ Found initial command "echo orange pineapple apple" in history file[0m
+[33m[stage-3] [0m[92m✓ Found initial command "echo pineapple apple mango" in history file[0m
+[33m[stage-3] [0m[92m✓ Found new command "echo pineapple banana pear" in history file[0m
+[33m[stage-3] [0m[92m✓ Found new command "echo grape pear" in history file[0m
+[33m[stage-3] [0m[92m✓ Found history command in history file[0m
+[33m[stage-3] [0m[92m✓ Found history append command in history file[0m
+[33m[your-program] [0m$ history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[92m✓ history -a command executed[0m
+[33m[stage-3] [0m[94mHistory file content after second append:[0m
```

Comment:
The logs for this are pretty verbose... Not sure how to fix atm but anecdotally users really don't like it when logs are too verbose, hard to know what to focus on